config/helm/chart/default/templates/Common/webhook/validatingwebhookconfiguration.yaml

```diff
@@ -27,15 +27,15 @@ webhooks:
       service:
         name: dynatrace-webhook
         namespace: {{ .Release.Namespace }}
-        path: /validate-dynatrace-com-v1beta2-dynakube
```

Comment:
don't "replace" the `v1beta2` validation
but add the `v1beta3` validation

Reply:
fixed in 38d79f13166e27799292383d0c9c3571f6a58be5